nextflow_schema.json

```diff
@@ -508,6 +508,52 @@
+                    "description": "Change the database version used for annotation."
+                },
+                "function_interproscan_db": {
+                    "type": "string",
+                    "help_text": "Use this to supply the path to a pre- downloaded InterProScan database. This can be any unziped InterProScan version. By default the database version '5.67-99.0' is downloaded. \n\nFor more details on where to find different InterProScan databases see tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/UserDocs.html#obtaining-a-copy-of-interproscan).\n",
+                    "description": "Path to pre-downloaded database."
+                },
+                "function_interproscan_applications": {
+                    "type": "string",
+                    "default": "PANTHER,ProSiteProfiles,ProSitePatterns,Pfam",
+                    "help_text": "Assigns the database /s to be used to annotate the coding regions annotated during the 'ANNOTATION' workflow of the pipeline. By default these include `PANTHER,ProSiteProfiles,ProSitePatterns,Pfam`.  \n- PANTHER (Protein ANalysis THrough Evolutionary Relationships): genes classified by their functions, using published scientific experimental evidence and evolutionary relationships. \n- PROSITE: protein domains, families, functional sites and specific patterns and profiles to identify them. \n- PFAM: protein families, represented by multiple sequence alignments and hidden Markov models (HMMs). \n\nThese databases were chosen based on the AMP workflow and therefore, with only these databases, do we guarantee the integration of the results to the AMPcombi final summary. \n\nNOTE: Currently, no integration of the results are implemented for the BGC and the ARG final summary tables. \n\nFor more information about all possible databases see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--applications`",
+                    "description": "Assigns the database /s to be used to annotate the coding regions."
+                },
+                "function_interproscan_enableprecalc": {
+                    "type": "boolean",
+                    "help_text": "This decreases the speed of functional annotation with InterProScan by pre-calculating matches found in the UniProtKB, thereby identifying unique matches in the query sequences for faster annotation. By default this is turned OFF.\n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `---diasable-precalc`",
```

Comment:
Do you mean decreases the speed or increasing? I assume it increases the speed if it's skipping a calculation step?

Reply:
🫢